realm/src/main/java/io/realm/RealmObject.java

```diff
@@ -78,9 +85,14 @@
     protected Row row;
     protected Realm realm;
 
+    private final List<RealmChangeListener> listeners = new CopyOnWriteArrayList<RealmChangeListener>();
```

Comment:
A marker interface wouldn't do much as it couldn't have any methods (as Standalone objects would have to implement them).